docs/MVS_parameters_list.csv

```diff
@@ -45,3 +45,5 @@
 43, None, The type of the component., demand, *demand*, str, None,type_asset,typeasset-label
 44, None," Input the type of OEMOF component. For example, a PV plant would be a source, a solar inverter would be a transformer, etc.  The `type_oemof` will later on be determined through the EPA.", sink, *sink* or *source* or one of the other component classes of OEMOF., str, None,type_oemof,typeoemof-label
 45, None, Unit associated with the capacity of the component.," Storage could have units like kW or kWh, transformer station could have kVA, and so on.", Appropriate scientific unit, str, NA,unit,unit-label
+46, None, The maximum amount of total emissions in the optimized energy system.,100000, None, Alphanumeric,None or kg/a,maximum_emissions,maxemissions-label
```

Comment:
```suggestion
46, None, The maximum amount of total emissions in the optimized energy system.,100000, None, Numeric,kg/a,maximum_emissions,maxemissions-label
```
The values should be restricted to positive ones.
How can the unit be `None`?
When defining new parameters, you would actually also define a default value here: https://github.com/rl-institut/multi-vector-simulator/blob/dev/src/multi_vector_simulator/utils/constants.py#L243

Reply:
seems like I was confused. I meant "Numeric or None", as the constraint can be switched off by `None`. I will adapt this and also adapt the parameter description of `maximumCap` as it says `None or float` for the unit.

Reply:
Hmm but now I'm confused concerning `unit` in the parameter definition. In in [constants.py](https://github.com/rl-institut/multi-vector-simulator/blob/71396b827086c5d9aa931fcc40f39564baef4493/src/multi_vector_simulator/utils/constants.py#L246) `unit` gives the type of the value while in `MVS_parameters_list.csv` it varies between type and unit of the parameter.

Reply:
see  #713